src/components/ResizeViewElement.tsx

```diff
@@ -0,0 +1,38 @@
+import { Dispatch, SetStateAction } from "react";
+import { twMerge } from "tailwind-merge";
+import Image from "next/image";
+
+type ResizeViewElementProps = {
+  value: number;
+  setValue: Dispatch<SetStateAction<number>>;
+  className?: string;
+};
+
+export function ResizeViewElement({
+  value,
+  setValue,
+  className,
+}: ResizeViewElementProps) {
+  return (
+    <div
+      className={twMerge(
+        "bg-primary w-24 flex flex-col items-center justify-between gap-4 rounded-lg text-2xl p-4 h-7, text-secondary",
+        className
+      )}
+    >
+      <button
+        className="py-2 text-4xl"
+        onClick={() => setValue((prev) => (prev < 100 ? prev + 1 : 0))}
```

Comment:
I'd put something like `handleIncreament` and `handleDecrement` above the return (because at first glance it hard to know what it does)
```suggestion
        onClick={handleIncreament}
```